src/controllers/AuthController.ts

```diff
@@ -1,6 +1,34 @@
-import { Response } from 'express'
+import { Request, Response, NextFunction } from 'express'
 import AuthService from '../services/AuthService'
 import { VerifyTokenRequest } from '../interfaces/AuthInterface'
+import { validator } from '../validation/validator'
+import { authSchema } from '../validation/authValidtaion'
+import GenericError from '../helpers/GenericError'
+
+const login = async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
check this comment https://github.com/CA-G12/mern-presently.io/pull/5#discussion_r996455766